cli/search/search.go

```diff
@@ -0,0 +1,67 @@
+package search
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/apex/log"
+)
+
+// isDeprecated checks if the program version is lower than 1.9.0.
```

Comment:
Looks like it checks if the program version is lower than 1.10.0

Reply:
Changed